ui/job-view/details/PinBoard.jsx

```diff
@@ -218,6 +219,61 @@ class PinBoard extends React.Component {
     }
   };
 
+  unclassifyAllPinnedJobsTitle = () => {
+    if (!this.props.isStaff) {
+      return 'Must be employee or sheriff';
+    }
+
+    if (!this.canCancelAllPinnedJobs()) {
+      return 'No jobs in pinboard';
+    }
+
+    return 'Unclassify all the pinned jobs';
+  };
+
+  canUnclassifyAllPinnedJobs = () => {
+    return (
+      this.props.isStaff && Object.values(this.props.pinnedJobs).length > 0
+    );
+  };
+
+  unclassifyAllPinnedJobs = async () => {
+    const {
+      notify,
+      currentRepo,
+      jobMap,
+      pinnedJobs,
+      recalculateUnclassifiedCounts,
+    } = this.props;
+
+    const { failureStatus } = await JobClassificationTypeAndBugsModel.destroy(
+      Object.values(pinnedJobs),
+      currentRepo,
+      notify,
+    );
+
+    if (!failureStatus) {
+      for (const pinnedJob of Object.values(pinnedJobs)) {
+        const job = jobMap[pinnedJob.id];
+
+        job.failure_classification_id = 1;
+
+        // Update the job to show that it's unclassified now.
+        const jobInstance = findJobInstance(job.id);
+        // Filter in case we are hiding unclassified.
+        if (jobInstance) {
+          jobInstance.refilter();
+        }
+      }
+      this.unPinAll();
+      window.dispatchEvent(new CustomEvent(thEvents.classificationChanged));
+      recalculateUnclassifiedCounts();
+    } else {
+      const message = `Error deleting classifications`;
```

Comment:
If you unpack `data`, I believe that will show the actual error message and then you can use it here.